src/services/login.js

```diff
@@ -32,6 +66,24 @@ export const facebookLogin = token =>
     },
   });
 
+export const facebookRefreshToken = async () =>
+  facebookLogin((await facebookStatus()).authResponse.accessToken);
+
+export const facebookStatus = async () => {
```

Comment:
style-wise, should we avoid mixing async-await and old school promises in the same code block?

Reply:
I don't see any other option. Facebook api don't support promise, so its not possible to use await. So we have to use callback. But we need to trigger a resolve.

Do you have an idea?

Reply:
Didn't know it did not support it...